saas/axops/src/ui/src/app/common/multiple-service-launch-panel/multiple-service-launch-panel.component.ts

```diff
@@ -335,7 +335,7 @@ export class MultipleServiceLaunchPanelComponent {
                     if (resubmitFailedParameters) { // resubmit failed
                         val = resubmitFailedParameters[property];
                     } else if (templateInputParams[property] && templateInputParams[property].hasOwnProperty('default')) {
-                        val = this.setParamValue(templateInputParams[property]['default']);
+                        val = this.setParamValue(templateInputParams[property]['default'], index);
```

Comment:
Why using `index` when you already have template? Please use template instead.

Reply:
Fixed.